src/parseNode.js

```diff
@@ -40,6 +40,8 @@ type ParseNodeTypes = {
         rowGaps: (?Measurement)[],
         hLinesBeforeRow: Array<boolean[]>,
         addEqnNum?: boolean,
+        maxNumRows?: 1,     // equation allows only 1 row.
+        maxNumCols?: 1 | 2, // equation: 1, split: 2.
```

Comment:
```suggestion
```
`maxNumRows` is not used, i.e., not returned by `parseArray`, and `maxNumCols` can be removed with the change above.

Reply:
Done.